.travis.yml

```diff
@@ -18,11 +16,16 @@ jobs:
   include:
     - php: 7.2
       env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 7.3
+      env: LINTER_RUN=run
     - php: 7.4
       env: COVERAGE_FLAGS="--coverage-clover coverage/clover.xml"
 
 install:
-  - composer self-update
   - composer update --prefer-dist --no-progress --no-suggest --optimize-autoloader $COMPOSER_FLAGS
 
-script: vendor/bin/phpunit $COVERAGE_FLAGS
+script:
+  - if [ "$LINTER_RUN" = "run" ]; then php vendor/bin/php-cs-fixer fix --verbose --dry-run ; fi;
```

Comment:
I assume it will raise errors if it founds coding-style violations?

Reply:
Yup